website/data-storage/src/config/configuration.ts

```diff
@@ -26,7 +26,7 @@ export default () => ({
     },
   },
   healthcheck: {
-    port: parseInt(process.env.HEALTHCHECK_PORT || '3000', 10),
+    port: parseInt(process.env.HEALTHCHECK_PORT || '3001', 10),
```

Comment:
What do you think Instead of using `parseInt` use `Number` constructor? 

```javascript 
port: Number(process.env.HEALTHCHECK_PORT || '3001')
```
Because `parseInt` usually use when you have `string` type like:
```javascript
parseInt("40 some text")
parseInt("Some text 40")
```